docs/extending-minerva.rst

```diff
@@ -0,0 +1,68 @@
+Extending Minerva
+=================
+
+Creating a Minerva Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Minerva utilizes the Girder plugin system, so it will be worthwhile to familiarize yourself with their section on `Plugin Development <http://girder.readthedocs.org/en/latest/plugin-development.html>`_.
+
+For Minerva specific plugins, specifying "minerva" as a dependency within your plugin is required to ensure things load in the proper order. Take this example plugin configuration:
```

Comment:
"Take this example plugin configuration for a new Girder plugin that extends the functionality of Minerva:"